content/en/news/releases/1.12.x/announcing-1.12.2/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Announcing Istio 1.12.2
+linktitle: 1.12.2
+subtitle: Patch Release
+description: Istio 1.12.2 patch release.
+publishdate: 2022-01-10
+release: 1.12.2
+aliases:
+    - /news/announcing-1.12.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** privileged flag to Istio-CNI Helm charts to set `securityContext` flag.
+  ([Issue #34211](https://github.com/istio/istio/issues/34211))
+
+- **Fixed** an issue where enabling tracing with telemetry API will cause malformed host header being used at the trace report request.
+  ([Issue #35750](https://github.com/istio/istio/issues/35750))
+
+- **Fixed** `istioctl pc log` command label selector not select default pod.
+  ([Issue #36182](https://github.com/istio/istio/issues/36182))
+
+- **Fixed** omitted setting `.Values.sidecarInjectiorWebhook.enableNamespacesByDefault` in the default revision
+mutating webhook and added --auto-inject-namespaces flag to `istioctl tag` controlling this setting.
+  ([Issue #36258](https://github.com/istio/istio/issues/36258))
+
+- **Fixed** the incorrect format of version and revision in the build info.
+  ([Pull Request #36409](https://github.com/istio/istio/pull/36409))
+
+- **Fixed** an issue where stale endpoints can be configured when a service gets deleted and created again.
+  ([Issue #36510](https://github.com/istio/istio/issues/36510))
+
+- **Fixed** an issue that sidecar iptables will cause intermittent connection reset due to the out of window packet.
+Introduced a flag `meshConfig.defaultConfig.proxyMetadata.INVALID_DROP` to control this setting.
+  ([Issue #36489](https://github.com/istio/istio/issues/36489))
+
+- **Fixed** `operator init --dry-run` creates unexpected namespaces.
+  ([Pull Request #36570](https://github.com/istio/istio/pull/36570))
+
+- **Fixed** an issue where setting `includeInboundPorts` with helm values does not take effect.
+  ([Issue #36644](https://github.com/istio/istio/issues/36644))
+
+- **Fixed** endpoint slice cache memory leak
+  ([Pull Request #36518](https://github.com/istio/istio/pull/36518))
+
+- **Fixed** changes in delegate virtual service do not take effect when RDS cache enabled
+  ([Issue #36525](https://github.com/istio/istio/issues/36525))
+
+- **Fixed** an issue when using Envoy [`v3alpha`](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.20.0#incompatible-behavior-changes) APIs in `EnvoyFilter`s.
```

Comment:
I haven't looked at this one in depth but looks like @howardjohn fixed it, but the mentioned issue is still open.

Reply:
closed it, thanks